packages/block-library/src/image/edit.native.js

```diff
@@ -307,11 +296,18 @@ export class ImageEdit extends React.Component {
 	}
 
 	render() {
-		const { attributes, isSelected, image } = this.props;
+		const { attributes, isSelected, image, imageSizes } = this.props;
 		const { align, url, height, width, alt, href, id, linkTarget, sizeSlug } = attributes;
 
 		const actions = [ { label: __( 'Clear All Settings' ), onPress: this.onClearSettings } ];
 
+		const getImageSizeOptions = () => map( imageSizes, ( { label, slug } ) => {
```

Comment:
I don't think it's necessary to make that a function here, just computing size options should be fine:
```
const sizeOptions = imageSizes.map( ( { label, slug } ) => ( { value: slug, label } ) );
```

Reply:
Good call, fixed here: https://github.com/WordPress/gutenberg/pull/19232/commits/8bd9ec1071269ba8f99c78600c7942db85e7be4b